indico/web/client/styles/themes/indico.scss

```diff
@@ -407,6 +407,7 @@ ul.day-list {
   .privacy-dropdown {
     display: inline-block;
     position: relative;
+    outline: none;
```

Comment:
why do we need/want to remove the outline here?

Reply:
Safari adds this weird blue outline when you focus on elements:
![image](https://user-images.githubusercontent.com/27357203/191706054-d52b3cfe-f7e6-4a14-be79-e360874b62a7.png)
And this doesn't affect anything on other browsers afaik

Reply:
Should I add a comment there specifying this is for Safari-specific behaviour?